docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -4,11 +4,11 @@ description: Privacy
 # Privacy
 
 Privacy in Pantheon refers to the ability to keep transactions private between the involved parties. 
-Other parties cannot access the transaction content, sending party, or list of participating parties. 
+Other parties cannot access the transaction content, sending party, or list of participating parties.
 
 !!! important
     For production systems requiring private transactions, we recommend using a network 
-    with a consensus mechanism supporting transaction finality. For example, [IBFT 2.0](../../HowTo/Configure-Pantheon/Consensus-Protocols/IBFT.md). 
+    with a consensus mechanism supporting transaction finality. For example, [IBFT 2.0](../../HowTo/Configure-Pantheon/Consensus-Protocols/IBFT.md). All private transaction parties must be online for a private transaction to be successfully distributed. If any parties are offline when the private transaction is submitted, the transaction fails and must be resubmitted.
 
```

Comment:
Technically the transaction will not be accepted, as in the transaction will never be attempted.

Reply:
Here are a few ways we might try to say this:

- The transaction is not attempted 
- The transaction is not done
- The transaction is rejected

Does the last option make sense technically?

Reply:
> The transaction is not attempted
Makes the most sense.

Reply:
I need to test this!

Reply:
Ok - this is correct.

"The transaction is not attempted" is the most appropriate.

Reply:
Great. I added that phrase to these updates.